src/main/java/org/terasology/navgraph/BitMap.java

```diff
@@ -83,16 +94,32 @@ public void getSuccessors(int offset, List<Integer> successors) {
         }
     }
 
+    /**
+     * Tells you whether there is any overlap between 'this' (the one that called the function) map and 'other' map
+     * @param other The map with which you want to check overlap
+     * @return true if there is any overlap
+     */
+
     public boolean overlap(BitMap other) {
         BitSet temp = (BitSet) map.clone();
         temp.and(other.map);
         return temp.cardinality() > 0;
     }
 
+    /**
+     * Merges 'this' map and 'other' and sets its value to 'this'
+     * @param other Bitmap with which you want to merge
+     */
+
     public void merge(BitMap other) {
         map.or(other.map);
     }
 
+    /**
+     * Gives information about the map with 'X's representing Walkable Blocks and ' 's representing blocked areas
```

Comment:
`'  's` looks weird. Is it literally a space stored in the map? If it is, maybe fully write that out, rather than typing it like this.